content/app_competition_rules.tex

```diff
@@ -47,18 +47,34 @@ \subsection{Qualification Procedure and Code Usage}
 
 \subsection{Game Structure}
 
-The clock stops during stoppages of play (such as ready and set state after goals) from the quarter-finals onward.  In round robin pool play, a game can finish in a draw as no penalty shoot-out will follow. In the promotion round, intermediate round, quarter finals, semi finals, 3rd place or final, a game that ends in a draw will be followed by a penalty shoot-out (see \cref{sec:penalty_shoot-out}).
+The clock stops during stoppages of play (such as ready and set state after goals) from the knock-out competition onwards.  In round-robin pool play, a game may finish in a draw and there will be no penalty shoot-out. Where a game cannot finish in a draw, including the knockout-competition (quarter finals, semi finals, 3rd place or final) and the promotion games, a game that ends in a draw will be followed by a penalty shoot-out (see Section~\ref{sec:penalty_shoot-out}).
 
-\subsection{Winner and Rankings}
-\label{sec:rankings}
+\subsection{Champions Cup and Challenge Shield}
+\label{sec:twoCompetitions}
+
+To provide better matched games for teams of all abilities the RoboCup Standard Platform League shall be divided into two separate competitions: the Champions Cup for the strongest teams and the Challenge Shield for all other teams. Champions Cup and Challenge Shield teams compete separately in each tournament. Teams are promoted to (or relegated from) the Champions Cup during the tournament. 
```

Comment:
```suggestion
To provide better matched games for teams of all abilities the RoboCup Standard Platform League shall be divided into two internal leagues: League 1 for stronger teams and League 2 for all other teams. In both leagues teams compete separately in each league during the tournament. Teams from both leagues are promoted to or relegated from League 1 after all games in each league have been played and the promotion games take place. 
```
Regarding naming: I thought leagues would also be a good phrase. Like we have it in Germany or England. And I just see an issue with the promotion games. Because normally the last games played at RoboCup are the final games. This would mean that final games of League 2 would have to be executed much earlier to allow the promotion games to be executed before Final games of League 1 happen. Or they will play inbetween, somehow.

Reply:
I'm not going to change the names yet, without general agreement, so I've left this commit suggestion for now.

RE promotion game times: Once the bottom CC teams, and finalist CS teams are known the promotion games can really be scheduled whenever. The proposed changes have 5-6 more games than the current structure, but don't required the 3 entire rounds of play before the knockout. I would think it feasible to hold promotion games from the 2nd last day onwards (and on the final day in-between the generally scheduled final games). This also gives the bottom placed CC teams something to prepare for on the finals day(s). However, the final scheduling is up to the OC as games can be fit in.